fe/src/project/Account/areas/Accounts/components/RegisterForm/RegisterForm.js

```diff
@@ -311,13 +312,26 @@ const RegisterForm = (props) => {
                                     error={(errors.companyPostalCode.length > 0) && dirty.companyPostalCode} />
                             </div>
                         </div>
+                        <div className="field">
+                            <NoSsr>
+                                <FormControlLabel 
+                                    control={
+                                        <Checkbox 
+                                            checked={acceptedTerms}
```

Comment:
**suggestion:** we should move it to the stateSchema

Reply:
okey